### ISSUE TITLE
Sample Interval greater 60s yields invalid results

### ISSUE THREAD

**Issue by Chrigux** [newrelic/nri-postgresql#166]
Sample Interval greater 60s yields invalid results. All 'Per Second' metrics are calculated as 0.

## Description
The interval in the postgresql-config.yml leads to invalid metric if configured greater than '60s'. If done so, all 'Per Second' metrics are calculated as 0 and are not usable anymore, for example in the Postgresql-Dashboard.

## Expected Behavior
The interval can be set to a value greater than 60s, like 120s or 300s, and still produce valid and usable metrics.

## Troubleshooting:
- Metrics like 'Buffers Allocated Per Second' in the 'PostgresqlInstanceSample' rely on the storer in the persist package from the infra-integration-sdk [Storer.go](https://github.com/newrelic/infra-integrations-sdk/blob/v3/persist/storer.go) to store the last retrived value to calculate the current rate.
- **Problem:** The storer only uses the DefaultTTL for data retention which is set to 1 minute. [storer.go:20](https://github.dev/newrelic/infra-integrations-sdk/blob/caaf06104c09510ee937e28304fa2728981014f3/persist/storer.go#L20)
- There is no error in the logs because en empty data retrieval from the storer gets interpreted as the 'first' revival after starting the agent.
[metrics.go:161](https://github.dev/newrelic/infra-integrations-sdk/blob/caaf06104c09510ee937e28304fa2728981014f3/data/metric/metrics.go#L161)

## Steps to Reproduce
1. Set 'interval" in  postgresql-config.yml to 120s.
2. Wait 5 minutes.
3. Look at the reported PostgresqlInstanceSample's in the querry builder on the newrelic website. `SELECT *  FROM   PostgresqlInstanceSample`
4. Confirm that the 'Per Second' values are all 0 and do not represent the actual conditions of the database monitored.

**Comment by sigilioso** [comment]
Hello @Chrigux,

Thanks for your contribution, we really appreciate it. This seems to be affecting to metrics whose type is `Delta`, we'll prioritize this issue and work on a fix as soon as possible.

**Comment by Chrigux** [comment]
@sigilioso can you show anything about the prioritization? We plan to monitor hundreds of PG databases in the future but the 60s limit produces too much data.

**Comment by sigilioso** [comment]
Hello @Chrigux , thanks for following up with this issue. We have scheduled it into our development pipeline and are planning to address it in the coming weeks.

**Comment by kang-makes** [comment]
Hi @Chrigux,

We made a change in the integrations-sdk to fix this not only on this integration but all on host integrations. Now `cacheTTL` has been raised to 6 minutes and a user is able to set the cache to an arbitrary interval if needed: https://github.com/newrelic/infra-integrations-sdk/pull/315/files#diff-798b0b0d816731857dfc9264d22e7d039de831672db4c3fe71554bde1825d9ebR27

The PR to upgrade the dependency (#173) to the SDK has been merged.

The release is automatic: https://github.com/newrelic/nri-postgresql/blob/4fed674044abf80650f391c2de83aa08d3c50f6d/.github/workflows/automated_release.yaml#L6C14-L6C24

The pipeline will create a prerelease today at 3pm (UTC) and promote last week prerelease to release so expect that an update will be ready in two weeks if canaries do not pop any issue.

Thank for the time taken to report this issues and the patience while we fixed it :D
